app/src/main/res/values-az/strings.xml

```diff
@@ -20,6 +20,13 @@
     <string name="unpin_conversation">Unpin</string>
     <string name="forward_message">Forward</string>
     <string name="compress_error">Unable to compress image to selected size</string>
+    <!-- vCard-->
+    <plurals name="and_other_contacts">
+        <item quantity="one">and %d other</item>
+        <item quantity="other">and %d others</item>
+    </plurals>
+    <string name="contact_info">Contact info</string>
+    <string name="add_contact">Add contact</string>
```

Comment:
replace contact_info with contact_details and I also added add_contact. I added both new strings into Simple Commons, it feels like strings like this could be needed in multiple apps.